MapboxAndroidDemo/src/global/java/com/mapbox/mapboxandroiddemo/MainActivity.java

```diff
@@ -244,24 +247,8 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     // Item click listener
-    ItemClickSupport.addTo(recyclerView).setOnItemClickListener(new ItemClickSupport.OnItemClickListener() {
-      @Override
-      public void onItemClicked(RecyclerView recyclerView, int position, View view) {
-        ExampleItemModel model = adapter.getItemAt(position);
-
-        // in case it's an info tile
-        if (model != null) {
-          if (showJavaExamples) {
-            startActivity(model.getJavaActivity());
-          } else {
-            startActivity(model.getKotlinActivity());
-          }
-
-          analytics.clickedOnIndividualExample(getString(model.getTitle()), loggedIn);
-          analytics.viewedScreen(getString(model.getTitle()), loggedIn);
-        }
-      }
-    });
+    itemClickSupport = ItemClickSupport.addTo(recyclerView);
+    itemClickSupport.setOnItemClickListener(this);
```

Comment:
NIT: Since we are always setting the listener in the `onResume`, this is redundant.